test/integration/sanity/Sanity.e2e.test.js

```diff
@@ -23,18 +28,22 @@ describe('Preview Sanity', () => {
             cy.getByTestId('bp-content').find('.bp-loading-wrapper').should('be.visible');
             cy.getPreviewPage(1);
             cy.contains('The Content Platform for Your Apps');
+            cy.get('@getFileInfo.all').should('have.length', 1);
```

Comment:
This is kinda cool, allows you to verify the invocation count of the matched route